var/spack/repos/builtin/packages/ufs-weather-model/package.py

```diff
@@ -32,54 +33,154 @@ class UfsWeatherModel(CMakePackage):
+        default=False,
+        description="Enable CCPP_32BIT (single precision arithmetic in slow physics)",
+    )
+    variant("debug", default=False, description="Enable DEBUG mode", when="@develop")
+    variant(
+        "debug_linkmpi",
+        default=True,
+        description="Enable linkmpi option when DEBUG mode is on",
+        when="@develop",
+    )
+    variant("inline_post", default=False, description="Enable inline post")
+    variant("multi_gases", default=False, description="Enable multi gases in physics routines")
+    variant("moving_nest", default=False, description="Enable moving nest code", when="@develop")
+    variant("openmp", default=True, description="Enable OpenMP")
+    variant("pdlib", default=False, description="Enable PDLIB (WW3)", when="@develop")
+    variant("parallel_netcdf", default=True, description="Enable parallel NetCDF")
+    variant(
+        "jedi_driver",
+        default=False,
+        description="Enable JEDI as top level driver",
+        when="@develop",
+    )
+    variant(
+        "cmeps_aoflux",
+        default=False,
+        description="Enable atmosphere-ocean flux calculation in mediator",
+        when="@develop",
+    )
+    variant(
+        "ccpp",
+        default=True,
+        description="Enable the Common Community Physics Package (CCPP)",
+        when="@:2.0.0",
     )
     variant(
         "ccpp_suites",
         default="FV3_GFS_v15p2,FV3_RRFS_v1alpha",
         description="CCPP suites to compile",
         values=("FV3_GFS_v15p2", "FV3_RRFS_v1alpha", "FV3_GFS_v15p2,FV3_RRFS_v1alpha"),
         multi=True,
+        when="@:2.0.0",
+    )
+    dev_ccpp_default = [
+        "FV3_GFS_v16",
+        "FV3_GFS_v16_flake",
+        "FV3_GFS_v17_p8",
+        "FV3_GFS_v17_p8_rrtmgp",
+        "FV3_GFS_v15_thompson_mynn_lam3km",
+        "FV3_WoFS_v0",
+        "FV3_GFS_v17_p8_mynn",
+        "FV3_GFS_v17_p8_ugwpv1",
+    ]
+    variant(
+        "ccpp_suites",
+        default=",".join(dev_ccpp_default),
+        description="CCPP suites to compile",
+        multi=True,
+        when="@develop",
     )
-    variant("inline_post", default=False, description="Compile post processing inline")
-    variant("multi_gases", default=False, description="Enable multi gases in physics routines")
-    variant("openmp", default=True, description="Enable OpenMP")
-    variant("parallel_netcdf", default=True, description="Enable parallel I/O in netCDF")
     variant(
         "quad_precision",
         default=False,
         description="Enable quad precision for certain grid metric terms in dycore",
+        when="@:2.0.0",
     )
+    variant("mom6solo", default=False, description="Build MOM6 solo executable", when="@develop")
+
     variant(
-        "simdmultiarch", default=False, description="Enable multi-target SIMD instruction sets"
+        "cmake_platform",
+        default="auto",
+        description="Override CMAKE_Platform ('linux.intel', 'hera.gnu', 'acorn', etc.)",
```

Comment:
Doesn't this depend on your OS? Like, a user can't choose this themselves, right? If you're on Linux and using the Intel compiler, you aren't allow to use any other choice. It might be better to auto-detect this always and remove the variant.

Reply:
done